README.md

```diff
@@ -1,2 +1,2 @@
 # cli-validation
-Library to help with flag validation of urfave/cli
+Library to help with flag validation of urfave/cli(current for v3 only). This ensures that urfave/cli codebase remains compact and clean of optional features
```

Comment:
An example of what flag validation is would be nice. Is it like type checking, range checking? If `cli` still handles some basic checks, maybe it worth mention which.

Reply:
@abitrolly Updated README.md to reflect what this library does. Let me know if that clarifies.

Reply:
That's a very good explanation. Thanks.